Milliner/src/Service/MillinerService.php

```diff
@@ -87,6 +89,15 @@ public function saveNode(
                 $jsonld_url,
                 $token
             );
+        } elseif ($event_type == "Version") {
+            try {
+                return $this->createVersion(
+                    $urls['fedora'],
+                    $token
+                );
+            } catch (Exception $e) {
+                $this->log->error('Caught exception: ', $e->getMessage(), "\n");
+            }
```

Comment:
Seeing this along with my previous thoughts about the header, I think I am partial to your thought of creating a new `/version/{uuid}` endpoint on Milliner

Reply:
Ok, I can work on creating a new Milliner endpoint instead of the header route. I'll modify the Alpaca PR accordingly (https://github.com/Islandora-CLAW/Alpaca/pull/61)

Reply:
Ok, I'll work on a separate Milliner endpoint and modify the Alpaca PR accordingly

Reply:
see 9173f40